packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -571,7 +571,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         type: ContainerMessageType,
         content: any,
         localOpMetadata: unknown,
-        opMetaData: unknown,
+        opMetadata: Record<string, unknown> | undefined,
```

Comment:
'opMetadata' must be constrained to "map-like objects" to ensure we can merge.

(Also, opMetaData -> opMetadata)